addons/base_automation/models/base_automation.py

```diff
@@ -61,6 +61,9 @@ class BaseAutomation(models.Model):
     filter_domain = fields.Char(string='Apply on', help="If present, this condition must be satisfied before executing the action rule.")
     last_run = fields.Datetime(readonly=True, copy=False)
     on_change_fields = fields.Char(string="On Change Fields Trigger", help="Comma-separated list of field names that triggers the onchange.")
+    update_fields_id = fields.Many2many('ir.model.fields', string='Watched fields',
```

Comment:
Naming convention says that the field should end with `_ids`.
I would also pick a more suggestive adjective, like `trigger_field_ids`.

Reply:
Good suggestion, thanks!